release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -1825,6 +1825,8 @@ In the table below, features are marked with the following statuses:
 
 * When upgrading to a new {product-title} z-stream release, connectivity to routers might be interrupted as router Pods are updated. For the duration of the upgrade, some applications might not be consistently reachable. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1809665[*BZ#1809665*])
 
+* When upgrading to a new {product-title} with the default CNI network provider set to OVN Kubernetes, the upgrade fails and the cluster is left in an unusable state. (link:https://bugzilla.redhat.com/show_bug.cgi?id=https://bugzilla.redhat.com/show_bug.cgi?id=1854175[*BZ#https://bugzilla.redhat.com/show_bug.cgi?id=1854175*])
```

Comment:
* You need "cluster" or "release" or something after {product-title}
* The BZ link is all sorts of weird. Both the link and the text need to be fixed.